projects/playground/src/app/child-route/nested-child-route.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-child-route',
```

Comment:
probably due to copy paste, this has the same selector as the `ChildRouteComponent`